packages/packages.go

```diff
@@ -95,6 +102,46 @@ type QFEPackage struct {
 	Caption, Description, HotFixID, InstalledOn string
 }
 
+// CommandRunner will execute the commands and return the results of that
+// execution.
+type CommandRunner interface {
```

Comment:
Should this be in a more common location than `packages/packages.go`? Maybe `utils`?

Reply:
moved to utils